### ISSUE TITLE
Resolve confusion over difference between statute and election behavior.

### ISSUE THREAD

**Issue by jldodds** [cjerdonek/formal-rcv#14]
@robdockins  and I find the specification of the optimization where multiple voters to mismatch a bit with what is done in practice.

> If the total number of votes of the two or more candidates credited
> with the lowest number of votes is less than the number of votes
> credited to the candidate with the next highest number of votes, those
> candidates with the lowest number of votes shall be eliminated
> simultaneously and their votes transferred to the next-ranked continuing
> candidate on each ballot in a single counting operation.

On our reading, this means all voters with the single lowest number of votes should be considered for simultaneous removal. Essentially, don't bother with a tie-break if the next vote is higher than the sum of those votes, just eliminate them all instead

The Berkeley statute defines precisely what seems to be done in practice in SF:

> During the elimination stage of any round, in the event that any
> candidate has more votes than the combined vote total of all candidates
> with fewer votes, all the candidates with fewer votes shall be
> eliminated simultaneously, and those ballots advanced to the next-ranked
> continuing candidate. (Ord. 7038-NS § 1, 5/20/08)

This is all a bit meaningless in a sense, as we have already proved that all of these procedures will result in the same outcome for the election.

**Comment by cjerdonek** [comment]
Yes, I agree the SF language is poor there and should be read to mean the same as the Berkeley language.  From my understanding, you are correct that SF implements it that way.  An additional clarification is that it should be the "maximal" such group of candidates (though the Berkeley language can be read as implying this).
